rnndb/bus/pci.xml

```diff
@@ -489,6 +489,24 @@
 			<value value="2" name="2_5GT"/>
 		</bitfield>
 	</reg32>
+	<reg32 offset="0x080" name="WIDTH">
+		<bitfield low="0" high="7" name="UNK1_WIDTH"/>
```

Comment:
both bit ranges seem to be always the same, which means I didn't found any trace/card where the values differ. so no clue